client/my-sites/plan-price/index.jsx

```diff
@@ -66,17 +66,22 @@ export class PlanPrice extends Component {
 		if ( displayFlatPrice ) {
 			const smallerPrice = renderPrice( priceRange[ 0 ] );
 			const higherPrice = priceRange[ 1 ] && renderPrice( priceRange[ 1 ] );
-
+			const termText = translate( ' per month', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **55** times:
`translate( 'per month' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/test-tc-unit-tests)